### ISSUE TITLE
Windows Server (2019) Core-Install and CTRL-ALT-DEL

### ISSUE THREAD

**Issue by JSuenram** [Ylianst/MeshCentral#3452]
When sending CTRL-ALT-DEL to Windows Core Server installation, nothing happens.
So Login to Desktop is not possible. BUG?

![image](https://user-images.githubusercontent.com/68963439/149343747-8227fe33-0999-4525-889e-cf0fcb1245c7.png)

**Comment by krayon007** [comment]
To inject ctrl-alt-del, I use the MSFT SAS API, but that doesn't exist on Windows Core. In some windows server versions you can work around this by manually copying sas.dll, but I'm not sure if this will work on windows Core.

**Comment by JSuenram** [comment]
> To inject ctrl-alt-del, I use the MSFT SAS API, but that doesn't exist on Windows Core. In some windows server versions (which also don't have sas api) you can work around this by manually copying sas.dll, but I'm not sure if this will work on windows Core.

Ok I understand. Other Remote-Control-Software does not have this issue.... we will try to place sas.dll on the servers, however I guess this is not the final solution. Just a workaround....

**Comment by krayon007** [comment]
Yes, I'm in the middle of a rewrite using the desktop duplication API. I'll check to see if that has it's own way to inject ctrl-alt-del.

**Comment by dinger1986** [comment]
Works fine for me @si458

**Comment by si458** [comment]
@dinger1986 not this problem still exists, just tried it here, install server 2019 standard (without desktop ui)
you cant CTRL+ATL+DEL on the login screen, but as explained in other linked post, its because the SAS API isnt part of the server core editions anymore!

**Comment by dinger1986** [comment]
Ah ok, I thought I have used it but maybe on 2016
